### ISSUE TITLE
Authorize Lambda to access api with cognito and get token

### ISSUE THREAD

**Issue by KrifaYounes** [AnomalyInnovations/serverless-stack-com#269]
I have a react-js lambda who expose an API Gateway
My lambda need to access to an API with AWS-COGNITO without login/password 
I search a way to generate the TOKEN and to access to the TOKEN generated on my react-js Lambda
Someone have an idea ?

**Comment by jayair** [comment]
@KrifaYounes Which token are you talking about? Can you post a link to any docs for the token you are referring to?

**Comment by KrifaYounes** [comment]
I want to generate a Authorization token from my lambda with cognito and I need to propagade the authorization token to a other java API

![img_1935](https://user-images.githubusercontent.com/3875788/45756313-d9920180-bc20-11e8-8f86-9ca8901f6330.JPG)

**Comment by jayair** [comment]
@KrifaYounes Thanks for the details. Are you able to get the authorization token from the Cognito Pool?

**Comment by KrifaYounes** [comment]
No, I am not able to get it. There is a way to get it with serverless configuration ?
My lambda is a react app with express server 
Thanks

**Comment by jayair** [comment]
To be honest, I have never used it this way. So I can't be of much help :(
